modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -10364,6 +10364,7 @@ object-entry-value-exceeds-integer-field-allowed-size=Object entry value exceeds
 object-entry-value-exceeds-long-field-allowed-size=Object entry value exceeds long field allowed size.
 object-entry-value-exceeds-maximum-long-field-allowed-size=Object entry value exceeds maximum long field allowed size.
 object-entry-value-exceeds-minimum-long-field-allowed-size=Object entry value exceeds minimum long field allowed size.
+object-entry-value-falls-bellow-minimum-long-field-allowed-size=Object entry value falls bellow minimum long field allowed size.
```

Comment:
I have one doubt: you added one language key, but don't we need to delete the other one that you replaced?

Reply:
I forgot to delete the old one, doing it right now.